src/routes/Sensors/DataTable/index.test.tsx

```diff
@@ -3,17 +3,27 @@ import DataTable from '.';
 import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
 import { sensorDataMock } from 'src/mocks/sensorDataMock';
+import { spotterListMock } from 'src/mocks/spotterListMock';
 
 const mockStore = configureStore([]);
 
 const store = mockStore({
   spotters: {
-    list: [],
+    list: spotterListMock,
     sensorData: sensorDataMock,
     sensorDataLoading: false,
     spottersRequestLoading: false,
     error: null,
   },
+  settings: {
+    selectedSpotter: spotterListMock[0],
+    sofarApiToken: 'some token',
+    spotterDataStartDate: undefined,
+    spotterDataEndDate: undefined,
+    spotterNodeId: undefined,
+    decoder: undefined,
+    timestamp: undefined,
```

Comment:
let's use a more explicit name. `timestamp` will likely become confusing. Maybe `timezoneOption`? or something like that?

Reply:
or `timestampFormat` as you named it in the component